### DISCUSSION TITLE
【求助】我的移动端绳网怎么这样了？

### DISCUSSION THREAD

**Discussion by qiuledadada** [share121/inter-knot#1692] [其他]
![Screenshot_2024-08-12-16-10-10-85_a252b927494330c](https://github.com/user-attachments/assets/ed95f97a-bb44-4f81-bab1-95540f0b5d15)
如题，有人能帮忙解决吗？

**Comment by ZhuYuan-148** [comment]
这位市民，绳网登陆请使用x浏览器

**Comment by qiuledadada** [comment]
请问全称就是这个吗？

**Comment by qiuledadada** [comment]
已解决感谢治安官的帮助
